package.json

```
@@ -328,7 +328,7 @@
     "jest-canvas-mock": "^2.3.1",
     "jsdom": "^11.2.0",
     "koa": "^2.7.0",
-    "lavamoat": "^5.3.5",
+    "lavamoat": "6.1.0",
```

Was this meant to be pinned on this specific version?

```suggestion
    "lavamoat": "^6.1.0",
```